### PR TITLE
Added scrollable friends list, firestore, and fixed login issue

### PR DESCRIPTION
**Pull request by Netzu00**

### CODE REVIEW

File: .idea/compiler.xml
Comment (christopherWilliams98) [inline .idea/compiler.xml:1]: is this by intention? I believe you may have a similar issue to mine where the version was changed sneakily by your android studio.

File: app/debug.keystore
Comment (christopherWilliams98) [inline app/debug.keystore:1]: Is this the debug file generated on your laptop for the login issue? I don't think it is a good idea to keep it in this location in our files unless it is needed.

File: app/src/androidTest/java/com/github/scribeWizTeam/scribewiz/AuthActivityTest.kt
Comment (christopherWilliams98) [inline app/src/androidTest/java/com/github/scribeWizTeam/scribewiz/AuthActivityTest.kt:1]: I assume the TODO and commented code is left for another branch?

Comment (christopherWilliams98) [inline app/src/androidTest/java/com/github/scribeWizTeam/scribewiz/AuthActivityTest.kt:1]: Nice efficient use of FireBaseUI!

File: app/src/main/java/com/github/scribeWizTeam/scribewiz/Activities/FirebaseUIActivity.kt
Comment (christopherWilliams98) [inline app/src/main/java/com/github/scribeWizTeam/scribewiz/Activities/FirebaseUIActivity.kt:1]: This is a great addition!

**Review by christopherWilliams98 — APPROVED** [review]
This is some great code with some essential edits and additions, looks modular, commented and well robust. See comments of individual sections of my review for a few issues I raised up.

File: app/debug.keystore
Comment (Netzu00) [reply app/debug.keystore:1]: Yes, this key is required to sign in from any other computer than mine

File: app/src/androidTest/java/com/github/scribeWizTeam/scribewiz/AuthActivityTest.kt
Comment (Netzu00) [reply app/src/androidTest/java/com/github/scribeWizTeam/scribewiz/AuthActivityTest.kt:1]: Yes, it is left for a future sprint where I will write better tests for FirebaseUI and Firestore

**Pull request merged**